### ISSUE TITLE
Suggested substitutions for obsolete pcb components

### ISSUE THREAD

**Issue by zcspencer** [juckettd/RaspberryPiCM4Handheld7Inch#5]
Hello! Love the project!
As I was assembling my digikey shopping cart, I noticed two components that are marked in Digikey as obsolete.
Looking for substitutions, I could not find any drop in replacements for the following parts:
- [F980J107MMAAXE](https://www.digikey.com/en/products/detail/kyocera-avx/F980J107MMAAXE/5957626) (Tantalum Capacitor), 
- [MMA8451QR1](https://www.digikey.com/en/products/detail/nxp-usa-inc/MMA8451QR1/2508916) (Accelerometer)

Curious if there are any recommendations, substitutions, or workarounds?

Thank you!

Edit: It seems that MMA8451QR1 can still be found around the web. No luck at all for F980J107MMAAXE

**Comment by weirdgyn** [comment]
I think caps will not be a problem... NXP suggest FXLS8971CF or FXLS8974CF as replacement for MMA8451QR1.. they're not pin-2-pin compatibile.

**Comment by zcspencer** [comment]
Thank you for your response! So are you saying that I can safely omit the capacitor?

**Comment by weirdgyn** [comment]
Nope.. you have to find a replacement with the same specifications:

type: tantalum capacitor (molded)
package size: 0603
voltage rating: 6.3V
capacity: 100uF (100 micro faradays)
tollerance: 20%

but it is an easier task than replacing an IMU.

So far I cannot find anything identical but I just looked on global providers non on specific productors catalogs.

This's quite the same: TLJP107M006R5400 
it's a little larger being 0805 instead 0603... so you will have to edit the board to fit the new cap.
